README.md

```diff
@@ -1,126 +1,2 @@
-```
-
-Packer will do the following:
-* Create a temporary EC2 instance, configure it with shell/ansible/puppet/etc. scripts.
-* Create an AMI from the EC2
-* Delete the EC2
-
-__Notes__:
- * Packer deploys a new AMI to the AWS account specified by the AwsProfile
- * Subsequent builds may require the [-force](https://packer.io/docs/commands/build.html#force) flag
-
-### Image Accessability
-This project is setup to build publicly accessible images.  To change it to
-build private images please refer to the [packer documentation](https://packer.io/docs/builders/amazon-ebs.html)
-for `ami_users` and `snapshot_users`options.
-
-### Testing
-As a pre-deployment step we syntatically validate our packer json
-files with [pre-commit](https://pre-commit.com).
-
-Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
-execute the validations by running `pre-commit run --all-files`.
-
-### CI Workflow
-The workflow to provision AWS AMI is done using pull requests.
-Just make changes with PRs and when th PR is merged a packer build
-will kick off which will build the image and deploys it to AWS.
-
-Packer will do the following:
-* Create a temporary EC2 instance, configure it with shell/ansible/puppet/etc. scripts.
-* Create an AMI from the EC2
-* Delete the EC2
-
-__Note__: The image will automatically be named gitrepo-branch (i.e. MyRepo-master)
-
-### Versioning
-Versions are managed by git tags. When a tag is pushed travis will build
-an AMI for that tag. Tag builds are immutable for downstream dependencies.
-Once a tag build is generated the AMI for that build will never go away.
-
-__Note__: The image will automatically be named gitrepo-tag (i.e. MyRepo-v1.0.0)
-
-### Searching
-List the built images by using the AWS CLI:
-```
-aws ec2 describe-images --owners 867686887310 --filters Name=tag:Name,Values=my-test-image
-```
-
-### Removal
-Building an AMI will create the AMI and one or more snapshots for the AMI.  When deleting
-the AMI remember to also delete its snapshots. Use the provided [bash script](deregister_ami.sh)
-to remove the AMI and its snapshots.
-
-## Contributions
-Contributions are welcome.
-
-Requirements:
-* Install [pre-commit](https://pre-commit.com/#install) app
-* Clone this repo
-* Run `pre-commit install` to install the git hook.
-
-## Deployments
-Travis runs packer which temporarily deploys an EC2 to create an AMI.
-
-## Continuous Integration
-We have configured Travis to deploy updates.
-
-## Issues
-* https://sagebionetworks.jira.com/projects/IT
-
-## Builds
-We use travis CI to automatically build and deploy images. Setup a travis ci build
-and add the AWS deployment credentials to the travis environment variables.
-
-## Secrets
-* We use the [AWS SSM](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html)
-to store secrets for this project.
+# packer-base-shinyserver
```

Comment:
Why remove the readme? I think the information is helpful when building the ami, granted its true it can be found in the original template

Reply:
Sure, I thought the MO was to start a new AMI from the template but you're right it does not hurt so I'll put it back.